qiskit_braket_plugin/providers/braket_backend.py

```diff
@@ -66,8 +98,36 @@ def acquire_channel(self, qubit: int):
     def control_channel(self, qubits: Iterable[int]):
         raise NotImplementedError(f"Control channel is not supported by {self.name}.")
 
-    def run(self, run_input, **options):
-        pass
+
+    def run(self, run_input: Union[QuantumCircuit, List[QuantumCircuit]], **options) -> AWSBraketJob:
+
+        convert_input = [run_input] if type(run_input) is QuantumCircuit else [_input for _input in run_input]
+        circuits: List[Circuit] = list(convert_circuit(convert_input))
+        shots = options["shots"] if "shots" in options else 1024
+        tasks = []
+        try:
+            for circuit in circuits:
+                task: Union[LocalQuantumTask] = self._aws_device.run(
+                    task_specification=circuit,
+                    shots=shots
+                )
+                tasks.append(task)
+
+        except Exception as ex:
+            logger.error(f'During creation of tasks an error occurred: {ex}')
+            logger.error(f'Cancelling all tasks {len(tasks)}!')
+            for task in tasks:
+                logger.error(f'Attempt to cancel {task.id}...')
+                task.cancel()
+                logger.error(f'State of {task.id}: {task.state()}.')
+            raise ex
+
+        return AWSBraketJob(
+            job_id=tasks[0].id,  # TODO: if there is 2 circuits what job_id should be returned?
+            tasks=tasks,
+            backend=self._aws_device,
```

Comment:
```suggestion
            backend=self,
```
As braket job should accept Qiskit's `Backend` class, not Braket's `Device`

Reply:
I can add test like these but, if we use the example you have provided (with a H gate) the counts will be diferent with every execution.